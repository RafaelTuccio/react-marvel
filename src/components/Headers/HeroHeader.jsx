
import logo_menor from "../assets/logo_menor.svg";
import FormHero from "../FormHero.jsx/FormHero";

const HeroHeader = (props) => {
    let { heroHandle } = props
    return(
        <header>
            <img src={logo_menor} alt="" />
            <FormHero heroHandle={heroHandle}/>
        </header>
    );
}

export default HeroHeader;