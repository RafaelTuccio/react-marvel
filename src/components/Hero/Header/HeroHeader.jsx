import logo_menor from "../../assets/logo_menor.svg";


const HeroHeader = () => {
    
    return(
        <header className="hero-header">
            <img src={logo_menor} alt="logo marvel" />
        </header>
    );
}

export default HeroHeader;