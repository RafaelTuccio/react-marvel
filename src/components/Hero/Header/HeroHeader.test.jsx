import { render, screen } from '@testing-library/react';
import HeroHeader from './HeroHeader';


describe("<HeroHeader />", () => {
    it("deve renderizar normalmente", () => {
        const { debug } = render(<HeroHeader />)
        debug();
        expect(screen.getByRole("img", {name: "logo marvel"}));
        
        
    })

    it("deve dar match com o snapshot", () => {
        const { container } = render(<HeroHeader />);
        expect(container.firstChild).toMatchSnapshot();
    })
    
})