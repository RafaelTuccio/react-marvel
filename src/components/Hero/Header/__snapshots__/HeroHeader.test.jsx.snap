// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HeroHeader /> deve dar match com o snapshot 1`] = `
<header
  class="hero-header"
>
  <img
    alt="logo marvel"
    src="logo_menor.svg"
  />
</header>
`;
